### ISSUE TITLE
Primitive Duplicates

### ISSUE THREAD

**Issue by thehabes** [CenterForDigitalHumanities/deer-LR#113]
Not sure if it is DEER or DEER-LR, but after expand properties noted as primitives are duplicated and look like

```JSON
{
   "name":[
         "Bryan",
         "name":{
             "value":"Bryan"
          }
   ]
}
```

This does not affect the UX or interface negatively, but is incorrect in the data structure.  An arbitrary choice is made to select and use the first value, but those values may become different as updates happen.  We need to look into this.

**Comment by thehabes** [comment]
Waiting on release of hotfix PR https://github.com/CenterForDigitalHumanities/deer/pull/52

**Comment by thehabes** [comment]
WONTFIX, waiting for DEER.  See https://github.com/CenterForDigitalHumanities/deer/issues/53
